frontend/packages/client/src/components/Settings/NotificationSettingsSection/CommunitiesList.js

```diff
@@ -1,24 +1,45 @@
 import Blockies from 'react-blockies';
-import { useCommunityDetails } from 'hooks';
+import { useWebContext } from 'contexts/Web3';
+import { useCommunityDetails, useUserCommunities } from 'hooks';
 import { subscribeNotificationIntentions } from 'const';
 
 export default function CommunitiesList({
   communitySubscription,
   updateCommunitySubscription,
 }) {
+  const {
+    user: { addr },
+  } = useWebContext();
+  const { data: userCommunities, loading: isLoadingUserCommunities } =
+    useUserCommunities({
+      addr,
+      count: 100,
+      initialLoading: false,
+    });
+  const displayCommunities = isLoadingUserCommunities
```

Comment:
This is a list of all communities a user participates, if leanplum userAttributes doesn't have a community, it will render that community still but with notification turned off

Reply:
under Settings it should only display list of communities that user subscribed before, looks like previous code is correct. Any reason we are changing this to be all communities?

Reply:
Got it, I misunderstood the design, will change it back